index.html

```diff
@@ -57,6 +57,329 @@ <h2>I'm a Software Developer</h2>
+            <span><a href="#">See project</a></span>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div>
+        <div class="about">
+          <div class="about-text">
+            <h4>About me</h4>
+          </div>
+          <div class="about-description">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+          </div>
+          <div class="about-footer">
+            <div class="about-btn">
+              <span><a href="#">Get my resume</a></span>
+            </div>
+            <div class="about-border"></div>
+          </div>
+        </div>
+      </div>
+      <div>
+        <ul class="language-sec">
+          <li class="lang-container">
+            <div class="lang-img">
+              <img src="./pics/rec.svg" alt="rec" />
+              <h4>Language</h4>
+            </div>
+            <div>
+              <ul class="lang-list">
+                <li><a href="#">JavaScript</a></li>
+                <li><a href="#">Ruby</a></li>
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">CSS</a></li>
+              </ul>
+            </div>
+          </li>
+          <li class="lang-container">
+            <div class="lang-img">
+              <img src="./pics/rec2.svg" alt="rec" />
+              <h4>Frameworks</h4>
+            </div>
+            <div>
+              <ul class="lang-list">
+                <li><a href="#">React.js</a></li>
+                <li><a href="#">Ruby on Rails</a></li>
+                <li><a href="#">RSpec</a></li>
+                <li><a href="#">Capybara</a></li>
+              </ul>
+            </div>
+          </li>
+          <li class="lang-container">
+            <div class="lang-img">
+              <img src="./pics/rec3.svg" alt="rec" />
+              <h4>skills</h4>
+            </div>
+            <div>
+              <ul class="lang-list">
+                <li><a href="#">Database Management</a></li>
+                <li><a href="#">Version Control</a></li>
+                <li><a href="#">CLI</a></li>
+                <li><a href="#">API Design</a></li>
+                <li><a href="#">Web Design</a></li>
+              </ul>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section>
+      <div class="interest">
+        <div class="intrest-description">
+          <h5>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h5>
+        </div>
+        <form
+          class="contact"
+          action="https://formspree.io/f/xzbqggzq"
+          method="POST"
+        >
+          <div class="contact-input">
```

Comment:
On your page, there is a small gap on the right side of the screen consider fixing it as it has a bad impact on the user experience. Remember your design needs to meet the minimum width of 375px.

## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/228577722-f834bc01-8960-4ef4-afd5-d17e0dbf5781.png)